apps/extension/src/pages/register/name-password-hardware/index.tsx

```diff
@@ -39,10 +39,12 @@ export const RegisterNamePasswordHardwareScene: FunctionComponent<{
   const form = useFormNamePassword();
 
   const [connectTo, setConnectTo] = useState<string>("Cosmos");
-
-  const bip44PathState = useBIP44PathState(type === "ledger");
+  const bip44PathState = useBIP44PathState(true);
   const [isBIP44CardOpen, setIsBIP44CardOpen] = useState(false);
 
+  const [keystoneWay, setKeystoneWay] = useState<string>("USB");
+  const [isKeystoneUSB, setIsUSB] = useState(true);
```

Comment:
I think `setIsUSB` should be `setIsKeystoneUSB`

Reply:
fixed